Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml.cs

```diff
@@ -242,9 +242,7 @@ private void TrackException(Exception e)
             }
             Properties.Clear();
             RefreshPropCount();
-            // TODO: uncomment this when API will be added.
-            Crashes.TrackError(e, properties /*, App.GetErrorAttachments().ToArray() */);
-
+            Crashes.TrackError(e, properties, App.GetErrorAttachments().ToArray();
```

Comment:
How was this tested, it would not build without the last `)`.

Reply:
I think he made this change before heading to lunch with the plan of testing after lunch and missed it. We've done some amount of testing with this correct. He wants to do a little more testing.